_includes/docs-sidebar.html

```diff
@@ -0,0 +1,27 @@
+<div class = "sidebar">
+  <div class="sidebar-content">
+   <!-- Add new sections here -->
+   {% assign order = 
+      "Dev Setup,
+      Add to Team Page"
+    %}
+
+    {% assign order = order | strip_newlines | split: "," %}
+    {% for i in order %}
+      {% assign cur = i | lstrip | rstrip %}
+      {% assign sec = site.docs | where: "sidebar-title", cur | first %}
+      {% assign spaceless = cur | replace: " ", "-" %}
+        <label for="{{spaceless}}-button"> {{sec.sidebar-title}}</label>
+        <input type="checkbox" id="{{spaceless}}-button" {% if page.sidebar-title == sec.sidebar-title %}checked{% endif %}/>
+        <div id="{{spaceless}}">
+          <ol class="sidebar-items">
+            {% for item in sec.items %}
+              {% assign spacelessDown = item | replace: " ", "-" | downcase | remove: "," | remove: "." | remove: "!" | remove: "'" | remove: "/" | remove: "?" %}
```

Comment:
I'm pretty sure a lot of the text manipulation you're doing here can be supplanted by `{% assign spacelessDown = item | slugify %}` ([read more here](https://jekyllrb.com/docs/liquid/filters/)).